packages/sync-actions/src/utils/common-actions.js

```diff
@@ -11,17 +17,30 @@ import * as diffpatcher from './diffpatcher'
  * @param  {Object} options.diff - the diff object
  * @param  {Object} options.oldObj - the object that needs to be updated
  * @param  {Object} options.newObj - the new representation of the object
+ * @param {Boolean} options.shouldOmitEmptyString - a flag to determine if we should treat an empty string a NON-value
  */
-export function buildBaseAttributesActions({ actions, diff, oldObj, newObj }) {
+export function buildBaseAttributesActions({
+  actions,
+  diff,
+  oldObj,
+  newObj,
+  shouldOmitEmptyString,
```

Comment:
I welcome suggestions on good names to this.
Please use the thread below

Reply:
Two hard problems in Computer Science: cache invalidation and *naming things* 😄